packages/swing-store-lmdb/package.json

```diff
@@ -22,11 +22,13 @@
   "dependencies": {
     "@agoric/assert": "^0.3.6",
     "better-sqlite3": "^7.4.1",
-    "node-lmdb": "^0.9.4"
+    "node-lmdb": "^0.9.4",
+    "tmp": "^0.2.1"
   },
   "devDependencies": {
     "@agoric/install-ses": "^0.5.20",
     "@agoric/swing-store-simple": "^0.4.6",
+    "@agoric/xsnap": "^0.6.9",
```

Comment:
I'm a bit weirded out by this dependency, but I can see why it's expedient. I wouldn't mind seeing a `swing-store-lmdb/test/test-snapstore.js` which only talked to the store, and exercised all of load/save/delete/commit, without using xsnap.

Then the existing test becomes more of an integration test that confirms we can use it in the right way (which of course must use both snapstore and xsnap, and thus wants to live either here or in `xsnap/`, and thus induces the slightly-weird dependency in either one direction or the other). I guess I'm marginally in favor of it living in `xsnap/` (the swingstore feels "lower level" than xsnap), but it's a toss-up.

Reply:
I expect `xsnap` to migrate to endo, so I don't want any dependencies on `swing-store-lmdb`. hm.

Reply:
Would it make sense to move this integration test to `cosmic-swingset` or `solo`?

Reply:
This dependency makes me really uncomfortable, because it means all the supposedly standalone tools that use the LMDB swingstore would then be dependent on xsnap.

Reply:
I moved it to `solo`, which already depends on `@agoric/xsnap`.

_I tried to move it to `cosmic-swingset` but `yarn test` started compiling a bunch of go code, and I didn't want to go there._

Reply:
@michaelfig I moved some snapstore tests to `solo/test` in c6e76e0; ok with you?

Reply:
> ok with you?

Totally fine if you want an integration test that isn't chain-specific.

Reply:
Having the xsnap+swingstore integration test live in `packages/swingset/` could work too: swingset uses both, and expects them to work together.